ddl/etcd_worker.go

```diff
@@ -14,18 +14,42 @@
 package ddl
 
 import (
+	"fmt"
+	"sync"
 	"sync/atomic"
 	"time"
 
+	"github.com/coreos/etcd/clientv3"
 	"github.com/coreos/etcd/clientv3/concurrency"
 	"github.com/coreos/etcd/mvcc/mvccpb"
 	"github.com/juju/errors"
 	"github.com/ngaut/log"
 	goctx "golang.org/x/net/context"
 )
 
-// ChangeOwnerInNewWay is used for testing.
-var ChangeOwnerInNewWay = false
+// EtcdWorker is an interface for DDL to access the etcd.
+type EtcdWorker interface {
```

Comment:
Since it is an interface, it doesn't have to be etcd, etcd it's the implementation.

I think separate it into two interfaces is better.
One is `SchemaSyncer`, the other one is `OwnerManager`.